dlt/common/schema/schema.py

```diff
@@ -66,7 +66,7 @@ def from_dict(cls, d: DictStrAny) -> "Schema":
         # verify schema
         utils.validate_stored_schema(stored_schema)
         # add defaults
-        utils.apply_defaults(stored_schema)
+        stored_schema = utils.apply_defaults(stored_schema)
```

Comment:
fyi this utils function changes the stored schema itself, not sure if you maybe wanted to clone it?

Reply:
no, it should happen in place